coderedcms/static/coderedcms/css/codered-admin.css

```diff
@@ -255,6 +255,9 @@ input[type='checkbox']::before, input[type='radio']::before {
     .nav-main .footer-submenu {
         overflow: hidden;
     }
+    .footer .footer-submenu {
```

Comment:
Could this be added to the rule above (line 255) - just to keep the rules to a minimum?